pages/History.qml

```diff
@@ -1095,11 +1095,20 @@ Rectangle {
                             color: "transparent"
                             Layout.fillWidth: true
                             Layout.preferredHeight: 20
+                            state: "ready"
 
                             MoneroComponents.TextPlain {
                                 font.family: MoneroComponents.Style.fontRegular.name
                                 font.pixelSize: 15
                                 text: {
+                                    if (currentWallet.isTrezorBacked()){
```

Comment:
Consider to move this to `Component.onCompleted`

Reply:
do you mean just the `currentWallet.isTrezorBacked()` check?

Reply:
Nvm, let's keep it as is